src/apps/content-editor/src/app/views/Dashboard/components/QuickJumps/QuickJumps.js

```diff
@@ -23,15 +23,17 @@ export function QuickJumps(props) {
             {props.cardTitle}
           </AppLink>
 
-          <Url
+          <Link
+            underline="none"
+            color="secondary"
             href={props.docsLink}
             target="_blank"
             title={props.docsLink}
-            className={cx(styles.bodyText, styles.Docs)}
+            sx={{ mt: 1.5, color: "primary.main" }}
```

Comment:
You are overriding color with sx here but you are defining color="secondary". You can just define color="primary"